packages/migration-graph/tests/fixtures/ember/addon-template/.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
can we add all of these config files to the git ignore? do we actually need them to test against for the migration-graph? (i would assume no)